src/filters/JobFilter.php

```diff
@@ -193,7 +193,7 @@ public function searchSenders()
      */
     private function parseDatetime($value, $isEnd = false)
     {
-        $dt = DateTime::createFromFormat('Y-m-d\TH:i', $value);
+        $dt = DateTime::createFromFormat('Y-m-d\TH:i', (string) $value);
```

Comment:
does this fix the case where `$value` is `null`? If so, it might be better to catch that case and return null directly before calling `createFromFormat()`:

```php
if ($value === null) {
     return null;
}
```

Reply:
Yes. Suggestion applied.